npm/nwpolicy.go

```diff
@@ -146,6 +151,7 @@ func (npMgr *NetworkPolicyManager) DeleteNetworkPolicy(npObj *networkingv1.Netwo
 		err   error
 		ns    *namespace
 		allNs = npMgr.nsMap[util.KubeAllNamespacesFlag]
+		// timer = metrics.StartNewTimer()
```

Comment:
Why is this deleted?

Reply:
I was thinking of adding a metric for execution time for deleting network policies, ipsets, etc. later. It's super easy to add, so I'll just delete the comments for now.